### ISSUE TITLE
Undefined object in Explore section while using loki as datasource

### ISSUE THREAD

**Issue by JnMik** [grafana/grafana#18864]
**What happened**:
In the explore section, if I select Loki datasource I have an error. Pressing F5 reload the page and Loki is then working.

```
An unexpected error happened.
Details
TypeError: Cannot read property 'split' of undefined

    in t
    in div
    in div
    in t
    in Wa
    in div
    in div
    in t
    in Connect(t)
    in div
    in t
    in div
    in div
    in t
    in Connect(t)
    in t
    in div
    in div
    in div
    in t
    in t
    in div
    in t
    in Connect(t)
    in d
    in Unknown
    in Unknown
    in a
```

**What you expected to happen**:
Loki panel should load and be usable right away

**How to reproduce it (as minimally and precisely as possible)**:
Configure Loki as a datasource, and add another datasource to be able to play with the select box in the explore panel.

Go in the Explore panel. If Loki is pre-selected, select another datasource and then come back to Loki. The error will be full page.

**Environment**:
- Grafana version: 6.3.5 but was present on 6.3.4 as well.
- Data source type & version: Loki version 0.3.0 but was present on 0.1.0 as well.
- OS Grafana is installed on: Grafana running in docker

Anyone else experimenting this ? 
Thanks !

**Comment by aocenas** [comment]
@JnMik Thanks for reporting, yeah there is an obvious bug in there. This though seems to be fixed in master and thus will be fixed in 6.4 release which should be this or next week.

@torkelo Considering how close the 6.4 is does it make sense to fix this in 6.3 branch?

**Comment by JnMik** [comment]
Glad to here a fix is coming ! 
thanks guys

**Comment by aocenas** [comment]
Should be fixed already in 6.4.

**Comment by JnMik** [comment]
Yep it works, tried it yesterday :)

Thanks !!
